Tests/test_excinfo.py

```diff
@@ -69,7 +69,7 @@ def f():
             try:
                 raise ValueError(63)
             except t():  # not matching
-                Assert(False)
+                assert False
```

Comment:
Use `self.fail()` instead? Same comment in other places where you have `assert False`.